main/src/ca/uwaterloo/flix/language/phase/Deriver.scala

```diff
@@ -537,6 +539,40 @@ object Deriver extends Phase[KindedAst.Root, KindedAst.Root] {
       KindedAst.MatchRule(pat, guard, exp)
   }
 
+  /**
+    * Creates a Boxable instance for the given enum.
+    *
+    * {{{
+    * enum E[a] with ToString {
```

Comment:
Not sure why it says `with ToString` here because Boxable requires more than that.

Reply:
Copy-paste negligence again. Shame on me.